integration-cli/docker_api_containers_test.go

```diff
@@ -138,11 +138,7 @@ func TestContainerApiStartVolumeBinds(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	bindPath, err := ioutil.TempDir(os.TempDir(), "test")
-	if err != nil {
-		t.Fatal(err)
-	}
-
+	bindPath := path.Join("/tmp", fmt.Sprintf("test.%s", makeRandomString(10)))
```

Comment:
@duglin normally it _might_ work _a little bit_, but you're passing `c:\users\xxx\appdata\temp\test.123` to docker daemon.